Block/Checkout/Success.php

```diff
@@ -15,172 +15,98 @@
-     * @var \Magento\Checkout\Model\OrderFactory
+     * @var OrderFactory
      */
     protected $orderFactory;
 
 
     /**
-     * @var \Adyen\Payment\Helper\Data
+     * @var Data
      */
     protected $adyenHelper;
 
     /**
-     * @var \Magento\Store\Model\StoreManagerInterface
+     * @var StoreManagerInterface
      */
     protected $storeManager;
 
-    /**
-     * @var ChargedCurrency
-     */
-    protected $chargedCurrency;
-
     /**
      * Success constructor.
      *
-     * @param \Magento\Framework\View\Element\Template\Context $context
-     * @param \Magento\Checkout\Model\Session $checkoutSession
-     * @param \Magento\Sales\Model\OrderFactory $orderFactory
-     * @param \Magento\Framework\Pricing\Helper\Data $priceHelper
+     * @param Context $context
+     * @param Session $checkoutSession
+     * @param OrderFactory $orderFactory
+     * @param Data $adyenHelper
+     * @param StoreManagerInterface $storeManager
      * @param array $data
      */
     public function __construct(
-        \Magento\Framework\View\Element\Template\Context $context,
-        \Magento\Checkout\Model\Session $checkoutSession,
-        \Magento\Sales\Model\OrderFactory $orderFactory,
-        \Magento\Framework\Pricing\Helper\Data $priceHelper,
-        \Adyen\Payment\Helper\Data $adyenHelper,
-        \Magento\Store\Model\StoreManagerInterface $storeManager,
-        ChargedCurrency $chargedCurrency,
+        Context $context,
+        Session $checkoutSession,
+        OrderFactory $orderFactory,
+        Data $adyenHelper,
+        StoreManagerInterface $storeManager,
         array $data = []
     ) {
         $this->checkoutSession = $checkoutSession;
         $this->orderFactory = $orderFactory;
-        $this->priceHelper = $priceHelper;
         $this->adyenHelper = $adyenHelper;
         $this->storeManager = $storeManager;
-        $this->chargedCurrency = $chargedCurrency;
         parent::__construct($context, $data);
     }
 
     /**
-     * Return Boleto PDF url
-     *
-     * @return string
-     */
-    protected function _toHtml()
-    {
-        if ($this->isBoletoPayment()) {
-            $this->addData(
-                [
-                    'boleto_pdf_url' => $this->getBoletoPdfUrl()
-                ]
-            );
-        }
-        return parent::_toHtml();
-    }
-
-    /**
-     * Detect if Boleto is used as payment method
-     *
-     * @return bool
-     */
-    public function isBoletoPayment()
-    {
-        if ($this->getOrder()->getPayment() &&
-            $this->getOrder()->getPayment()->getMethod() == \Adyen\Payment\Model\Ui\AdyenBoletoConfigProvider::CODE) {
-            return true;
-        }
-        return false;
-    }
-
-    /**
-     * @return null|\string[]
-     */
-    public function getBoletoData()
-    {
-        if ($this->isBoletoPayment()) {
-            return $this->getOrder()->getPayment()->getAdditionalInformation('action');
-        }
-        return null;
-    }
```

Comment:
Restore Boleto as it should work without the GC

Reply:
In the success page the Boleto voucher is being shown by the bundle successfully. So I think we can make the payments with manual form as in v6 and show the success with the GC

Reply:
sounds great :)